user_guide_src/source/dbmgmt/forge.rst

```diff
@@ -249,8 +251,8 @@ $forge->addColumn()
 -------------------
 
 The ``addColumn()`` method is used to modify an existing table. It
-accepts the same field array as above, and can be used for an unlimited
-number of additional fields.
+accepts the same field array as :ref:`Creating Tables <adding-fields>`, and can
+be used for an unlimited number of additional fields.
```

Comment:
"unlimited" is limited based on Database Engine used. I think this can be changed to: "can be used to add a number of additional fields"

Reply:
Surely "unlimited" is hype.

Reply:
Done. Thanks!